test/scripts/generate-event-handlers-json.spec.ts

```diff
@@ -0,0 +1,60 @@
+import path from "path";
+import getDriver, { Driver } from "./generate-handlers-json.driver";
+
+const FIXTURE_PATH = path.join(__dirname, "./fixtures/$w-events.d.ts");
+
+describe("generate-events", () => {
```

Comment:
did you compare the output of the new code with the existing output of codemodel ?
let's do it manually once to make sure they're aligned.

Reply:
Yes, I did.. I saved in https://pastebin.com/C143k5Qt and compared them.